src/desktop/integration/DesktopIntegratorLinux.js

```diff
@@ -108,7 +108,13 @@ export function integrate(): Promise<void> {
 	const prefix = app.name.includes("test") ? "test " : ""
 	return copyIcons().then(
 		() => createDesktopEntry(prefix)
-	)
+	).then(() => {
+		if (process.env["XDG_CURRENT_DESKTOP"] !== "GNOME") return
+		try {
+			exec(`update-desktop-database "${path.join(app.getPath('home'), ".local/share/applications")}`)
```

Comment:
Can we use `execFile`? It doesn't spawn shell and doesn't process weird shell escape sequences.